ansible/roles/operator-pipeline/templates/openshift/tasks/github-add-comment.yml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-add-comment
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/categories: Git
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: github
+    tekton.dev/displayName: "add github comment"
+spec:
+  description: >-
+    This Task will add a comment to a pull request or an issue.
+    It can take either a filename or a comment as input and can
+    post the comment back to GitHub accordingly.
+  workspaces:
+    - name: github-bot-token
+    - name: comment-file
+      optional: true
+      description: The optional workspace containing comment file to be posted.
+
+  results:
+    - name: OLD_COMMENT
```

Comment:
nitpick: All the other results/ params in the pipelines are lowercase

Reply:
It is imported from https://github.com/tektoncd/catalog/tree/main/task/github-add-comment/0.4, still, want to update it?

Reply:
I'm confused...if the task is imported, why is the code also copy/pasted here?

Reply:
Because we are doing it same for yaml-lint and git-clone task.

Reply:
Only for the sake of the ansible deployments :slightly_smiling_face: they don't import the code from the internet, so we duplicate it.

Reply:
Could you also mention the installation of this task in README, under the existing comment
```
# Install external dependencies
```
?

Reply:
I already mentioned it in README, I think you can see that in changes.